client/my-sites/add-ons/hooks/use-add-ons.ts

```diff
@@ -19,10 +22,55 @@ export interface AddOnMeta {
 	icon: JSX.Element;
 	featured?: boolean; // irrelevant to "featureSlugs"
 	name: string | React.ReactChild | null;
+	quantity?: number;
 	description: string | React.ReactChild | null;
 	displayCost: string | React.ReactChild | null;
+	variations?: Partial< AddOnMeta >[];
 }
 
+const AVAILABLE_PRODUCTS = [ 50, 100 ];
+
+const buildStorageAddOn = (
```

Comment:
This method filters available options based on the current storage limit for a site. The implementation is using mocked data for now, I'll be updating the method to use real data from `useMediaStorageQuery `